src/module-analyzer.js

```diff
@@ -3,6 +3,7 @@ import {Container} from 'aurelia-dependency-injection';
 import {TemplateRegistryEntry} from 'aurelia-loader';
 import {ValueConverterResource} from 'aurelia-binding';
 import {BindingBehaviorResource} from 'aurelia-binding';
+import {ViewEngineHooksResource} from 'aurelia-binding';
```

Comment:
Feel free to collapse those into a single import. It's just an accidental oversight in this case.